### PR TITLE
Add user location to resources query

### PR DESCRIPTION
**Pull request by ZanderSparrow**

### CODE REVIEW

File: app/components/Resources/ResourcesTable.js [hunk 1]
```diff
@@ -98,16 +101,16 @@ class ResourcesTable extends Component {
 		let page = 0;
 		if(this.state.openFilter) {
 			this.setState({
-				page: page,
+				page,
 				resources: this.state.allResources,
-				currentPage: this.state.allResources.slice(page, page + 9),
+				currentPage: this.state.allResources.slice(page, resultsPerPage),
```
Comment (jfhamlin) [inline app/components/Resources/ResourcesTable.js]: If `page` were anything but zero, this wouldn't be correct.

File: app/components/Resources/ResourcesTable.js [hunk 2]
```diff
@@ -116,26 +119,20 @@ class ResourcesTable extends Component {
...
+			this.getLocationGoogle()
+			.then(loc => {
+				self.loadResourcesFromServer(loc);
+			});
+			
 		}
-
-		this.loadResourcesFromServer();
-
+		this.loadResourcesFromServer({lat: 1, lng: 1});
```
Comment (jfhamlin) [inline app/components/Resources/ResourcesTable.js]: Do we want to make this call? Both branches of the `if` above call `loadResourcesFromServer`.

Comment (ZanderSparrow) [inline app/components/Resources/ResourcesTable.js]: yeah, good call, that was left over from testing

File: app/components/Resources/ResourcesTable.js (hunk 1 above)
Comment (ZanderSparrow) [inline app/components/Resources/ResourcesTable.js]: page is explicitly reset to 0 at the beginning of the function. I changed it, but it seems like a lateral change to me.

Comment (jfhamlin) [inline app/components/Resources/ResourcesTable.js]: Yeah, it's a minor thing. My only qualm is that the expression looks like it happily takes a variable, `page`, but if that variable actually varied, the expression wouldn't work. We can leave it.

**Comment by jfhamlin** [comment]
LGTM

**Pull request merged**